src/System.IO.FileSystem/src/System/IO/File.cs

```diff
@@ -632,28 +632,12 @@ public static void Move(string sourceFileName, string destFileName)
 
         public static void Encrypt(string path)
         {
-            if (path == null)
-                throw new ArgumentNullException(nameof(path));
-
-            // TODO: Not supported on Unix or in WinRt, and the EncryptFile API isn't currently
-            // available in OneCore.  For now, we just throw PNSE everywhere.  When the API is
-            // available, we can put this into the FileSystem abstraction and implement it
-            // properly for Win32.
-
-            throw new PlatformNotSupportedException(SR.PlatformNotSupported_FileEncryption);
+            FileSystem.Encrypt(path);
```

Comment:
Better to do the null check here before calling `FileSystem.Encrypt` to help make sure we're keeping parameter checking consistent.

Reply:
done